src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,31 @@
+import * as vscode from 'vscode';
+
+export class ExportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You convert user's code written in any programming language to the MongoDB Shell syntax.
+Take a user prompt as an input string and translate it to the MongoDB Shell language.
+Keep your response concise.
+You should suggest queries that are performant and correct.
```

Comment:
I am not an expert in prompt crafting but that seems odd here - assuming the goal is to do an exact translation of the user's code, I would not expect for the model to try and improve the performance/correctness of the queries the user wrote.

Reply:
Maybe
```suggestion
Ensure the code is translated correctly.
```
? Or is should we remove it entirely? Hard to tell without trying it more.

Reply:
I used this part from the generic endpoint. Rhys was experimenting with accuracy and I guess this phrasing gives more relevant results. Currently, the export to language prompt performs well. I would not change it until we have accuracy tests in place and can see if changes improve or make it worse.

Reply:
I played more around and we do need to improve prompt. Currently, it performs well with syntax that matches a single shell command. But when I tried with a more complicated script it generated MongoDB node driver syntax and not shell.

Reply:
From which language were you translating from? We probably want to try a few.

Reply:
I tried with this snipper from java and it generated a nice MongoDB node driver code :) https://www.mongodb.com/docs/drivers/java/sync/v4.9/usage-examples/insertOne/#:~:text=To%20insert%20a%20document%2C%20construct,returns%20an%20instance%20of%20InsertOneResult%20.